### ISSUE TITLE
Bigger size of the gadget for 4K ?

### ISSUE THREAD

**Issue by Omar-Os** [LibreHardwareMonitor/LibreHardwareMonitor#1093]
Can you please let us have even bigger size than "Very Large" of the gadget? The "Very Large" is still not that good in 4K! Specially because I'm using a 4K 65" TV and sitting 2-3 meters away from it.

**Comment by VOOM108** [comment]
Same with a regularly sized 4k screen. Would very much like to see even bigger fonts... :)

**Comment by glynnie** [comment]
I think the gadget in the latest version can be resized. Is it possible to update the resolution of the icons as at 4k they are very blurred.

**Comment by nykoo** [comment]
yes it can be resize un width but not in height, and font size stay the same.
Font size as only 4 level of size but i's way too smal for 4k or small monitor with high resolution
apparently there is a pull request about this
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1788#issue-3226582132

edit: i had change value described in https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1788#issue-3226582132 to :

 switch (i)
            {
                case 0: size = 9f; name = "Small"; break;
                case 1: size = 11f; name = "Medium"; break;
                case 2: size = 22f; name = "Large"; break;
                case 3: size = 44f; name = "Very Large"; break;
                default: throw new NotImplementedException();
            }

and the recompiled the project for windows exe and now font size way bigger and perfect for my screen
